docs/configuration.rst

```diff
@@ -135,6 +135,18 @@ The log file will not be rotated automatically as this is problematic due to Ral
             notifempty              # don't attempt to rotate empty ones
     }
 
+If you are benchmarking source builds of Elasticsearch, Rally will write the log output of the build to ``~/.rally/logs/build.log``. Similarly to the configuration for ``rally.log``, you can use the following ``logrotate`` configuration as a starting point::
```

Comment:
When reading this first time, it may not be entirely clear what the logrotate config is for, I suggest we rephrase:

> you can use the following ``logrotate`` configuration as a starting point::

to

> you can use the following ``logrotate`` configuration to restrict the volume of the log file::

Reply:
I thought that can be inferred based on context (i.e. reading the paragraph above about `rally.log`).